### ISSUE TITLE
bug: my bind_to_monitor has no effect

### ISSUE THREAD

**Issue by jaymorelli96** [glzr-io/glazewm#560]
I have the following:
```
workspaces:
  - name: "1"
    bind_to_monitor: 1
  - name: "2"
    bind_to_monitor: 2
  - name: "3"
    bind_to_monitor: 2
  - name: "4"
    bind_to_monitor: 1
```

And it doesn't have any effect. If I am on monitor 1 and I change focus to workspace 3 it will open on the monitor 1. If I am on monitor 2 and I focus on workspace 3 it will open on monitor 2...

**Comment by Crackpotpower56** [comment]
Same here Workspace 1 keeps spawning on monitor 2 no matter what i do.

**Comment by gfvioli** [comment]
I kinda had a problem like this in which if I reassigned my main monitor to one of the external ones, the laptop monitor would always still be 0, so now it would share the binding with whichever monitor was the main one. I default back to keep my laptop monitor as the main monitor and it solved that problem.

However, I currently face the problem that no matter what I do, workspaces 4 and 5 bind to the which ever monitor workspace 3 is at. This is my current config:
```yaml
workspaces:
  - name: '1'
    display_name: 'Fun'
    bind_to_monitor: 1
    keep_alive: true
  - name: '2'
    display_name: 'Browser'
    bind_to_monitor: 0
    keep_alive: true
  - name: '3'
    display_name: 'Work'
    bind_to_monitor: 2
    keep_alive: true
  - name: '4'
    display_name: 'Messaging'
    bind_to_monitor: 0
    keep_alive: true
  - name: '5'
    display_name: 'Outlook'
    bind_to_monitor: 0
    keep_alive: true
  - name: '6'
  - name: '7'
  - name: '8'
  - name: '9'
```

if anybody has any tips, would be great to know.

**Comment by lars-berger** [comment]
`bind_to_monitor` has been fixed up in v3.1.1 👍
